### DISCUSSION TITLE
On Premise Installation

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [vueform/builder#2] [Questions]
I saw this, but I haven't found any information about how to install the builder on premise or if there is a difference in pricing. Can I just follow the wizard and skip the account process? How to set things up with the database?

![image](https://github.com/vueform/builder/assets/86342787/43dc0672-ca8c-4f63-81d1-c63d1f52861c)

**Comment by adamberecz** [comment]
The prices are for the on-premise version (the current online is free).

If you [sign up](https://app.vueform.com/signup), create a project and follow the installation guide, you'll get customized steps for the on-premise setup, which will already include an API Key.

If you start without an account with the Installation guide in our Docs, you'll be asked to create an account at some point and insert an API Key manually - that's the difference.

Which ever you start with you'll end up having an account, an API Key and an on-premise version of the Builder.

Hope this help.

**Reply by sschneider-ihre-pvs** [reply]
Ok, how does that work behind a corporate firewall?

**Reply by adamberecz** [reply]
API validation is required, which could be an issue with firewalls. We do however offer an on-prem version without remote calls. If that's something you might need, reach out to me at adam@vueform.com and we can discuss.
